spec/README-compat.md

```diff
@@ -0,0 +1,139 @@
+
+# Wasm Image Specification v0.0.0
+
+## Introduction:
+
```

Comment:
Can we add some text to the main spec about the availability of a compat format?

Reply:
OK agreed. will do.

Reply:
intentionally not to apply your suggestion in favor of added general info in README.md. PTAL